go/carmen/carmen.go

```diff
@@ -401,7 +400,7 @@ type BulkLoad interface {
 
 	// SetBalance sets the balance of the given account address
 	// to the input value.
-	SetBalance(Address, *big.Int)
+	SetBalance(Address, Amount)
```

Comment:
should it be here and in other methods a pointer to `Amount`? It is done so in Ethereum

Reply:
It makes sense in setters to accept pointers

Reply:
I do not think there is a benefit for passing it as a pointer. It might actually introduce additional memory allocations by Go to move the passed element to the heap.

Keys or values in other API functions are also 32-byte large and passed by value. We have not seen problems with this so far, and I would favor consistency.

Reply:
thanks for explanation